pandas/tests/frame/test_nonunique_indexes.py

```diff
@@ -322,15 +322,15 @@ def test_set_value_by_index(self):
 
         df = DataFrame(np.arange(9).reshape(3, 3).T)
         df.columns = list("AAA")
-        expected = df.iloc[:, 2]
+        expected = df.iloc[:, 2].copy()
```

Comment:
Copying those "expected" frames avoids any potential issues where `expected` might be modified(or not) through what is actually being tested